src/Simplify_Mul.cpp

```diff
@@ -80,6 +80,9 @@ Expr Simplify::visit(const Mul *op, ExprInfo *bounds) {
             rewrite(max(x, y) * min(x, y), x * y) ||
             rewrite(max(x, y) * min(y, x), y * x) ||
 
+            rewrite(x * select(y, 1, 0), select(y, x, 0)) ||
+            rewrite(x * select(y, 1, 0), select(y, x, 0)) ||
```

Comment:
These two appear to be the same. Was one supposed to be commuted?

Reply:
Yes. They somehow became the same rule when I ran it through filter_rewrite_rules and got me duplicating it..